lib/index.js

```diff
@@ -14,6 +14,8 @@ const LOG_DEBUG = 2
 const LOG_INFO = 1
 const LOG_WARNING = 0
 
+const boolOption = option => option === true ? '1' : '0'
```

Comment:
This actually interesting: I was planning on supporting `true` and `false` for MK but the C++ library was failing in one of macOS and Linux (do not remember which), so I pulled that.